docs/configuration.md

```diff
@@ -0,0 +1,140 @@
+## How to configure RIF Services for local development.
+
+When you run RConnect on Regtest with a local node you need to setup all the 
+neccesary infrastructure to allow the wallet to work with RIF Services. 
+To do so you need to configure some information about the API's that allow
+us to access those services.
+
+If we see the RConnect General Architecture Diagram:
+
+![General Diagram](./general-rconnect-architecture.png)
+
+There we have 1 RSK Node, 1 Lumino Explorer, 1 Lumino Hub and several Notifiers.
+All these are API's running somewhere, so we need to setup those to let the wallet
+to access them. 
+
+The file that let us configure this is [this one.](../app/scripts/controllers/rif/configuration/index.js)
+There we will see that is a JS file with some functions inside, basically what we need
+to change is the function `getInitialConfigStructure` that makes the initial setup.
```

Comment:
```suggestion
This is a JS file which defines a particular set of functions. We need to change the `getInitialConfigStructure` function because it is the one that does the initial setup.
```